hbase-server/src/main/java/org/apache/hadoop/hbase/wal/WALSplitUtil.java

```diff
@@ -345,16 +345,16 @@ public boolean accept(Path p) {
 
   /**
    * Move aside a bad edits file.
-   * @param walFS WAL FileSystem used to rename bad edits file.
+   * @param fs the file system used to rename bad edits file.
    * @param edits Edits file to move aside.
    * @return The name of the moved aside file.
    * @throws IOException
    */
-  public static Path moveAsideBadEditsFile(final FileSystem walFS, final Path edits)
+  public static Path moveAsideBadEditsFile(final FileSystem fs, final Path edits)
```

Comment:
Why this name changing? Because for HFile based split we will write HFile to the root fs instead of wal fs?

Reply:
Yes. For hfile, the passed fs is root file system.